cluster/versions.tf

```diff
@@ -1,9 +1,9 @@
 terraform {
   required_providers {
     aws = {
-      version = "~> 4.0"
       source  = "hashicorp/aws"
+      version = ">=4.0.0"
     }
   }
-  required_version = ">= 1.0.1"
+  required_version = ">= 0.14"
```

Comment:
Should use a more recent version

Reply:
have updated in other branch which will be merged before this